config/environments/production.rb

```diff
@@ -46,7 +46,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = ENV.fetch('FORCE_SSL', true) != 'false'
```

Comment:
Should it be `'true'`, since you are comparing it with a string?

Reply:
I thought about that, but then I was like "I want to make it *really hard* to turn off, so to turn off force SSL you have to set it to a string that says "false"; otherwise it's going to assume you need SSL.